### PR TITLE
Add locale column to videos table

### PR DESCRIPTION
**Pull request by bethanyaconnor**

### CODE REVIEW

**Review by nkiruka — APPROVED** [review]
LGTM!

**Review by Hamms — COMMENTED** [review]
What's the long-term plan for how we indicate that two videos are differently-localized equivalents?

**Comment by bethanyaconnor** [comment]
They will share the same key. This requires removing the uniqueness on that column, which will definitely be done in a different PR.

Spec: https://docs.google.com/document/d/1QesZPr0qUq340oRidlQ9WIlGG-fyTfb5tJCssZkwaos/edit#

**Comment by Hamms** [comment]
Any reason not to go ahead and add a unique index on `[:key, :locale]`, and update the `validates_uniqueness_of` call to include both now?

**Comment by bethanyaconnor** [comment]
Good call. I modified the uniqueness validation in the video model and added a unique index.

**Review by Hamms — APPROVED** [review]
:+1:

File: dashboard/app/models/video.rb [hunk 1]
```diff
@@ -16,7 +21,7 @@ class Video < ActiveRecord::Base
 
   default_scope {order(:key)}
 
-  validates_uniqueness_of :key
+  validates_uniqueness_of :key, scope: [:locale]
```
Comment (joshlory) [inline dashboard/app/models/video.rb:24]: 👍

**Review by joshlory — APPROVED** [review]
Super clean!

**Pull request merged**